src/viam/sdk/services/mlmodel/private/proto.cpp

```diff
@@ -64,10 +68,11 @@ ::grpc::Status pb_value_to_tensor_t(const MLModelService::tensor_info& tensor_in
             ++depth;
             std::for_each(
                 children.rbegin(), children.rend(), [&vs](const auto& v) { vs.push(&v); });
-        } else if (std::is_same<T, std::uint8_t>::value && vs.top()->has_string_value()) {
+        } else if (std::is_same<T, std::uint8_t>::value &&
+                   (vs.top()->kind_case() == gp::Value::kStringValue)) {
             const auto& sv = vs.top()->string_value();
             std::string decoded;
-            if (!::google::protobuf::Base64Unescape(sv, &decoded)) {
+            if (!absl::Base64Unescape(sv, &decoded)) {
```

Comment:
Here is where we have switched to using the absl Base64 routines instead of the protobuf ones, which are removed (recently: only yesterday it stopped working for me after a brew upgrade).